packages/create-playwright/assets/examples/5-pom-fixtures.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { test, expect } from './5-pom-fixtures';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('https://todomvc.com/examples/vanilla-es6/');
```

Comment:
Let's do `test.use({ baseURL })` and put `page.goto('./')` to every test instead? Also, what happened to `TodoPage.goto()` method?